scripts/breaking_changes_checker/checkers/added_method_overloads_checker.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import sys
+import os
+sys.path.append(os.path.abspath("../../scripts/breaking_changes_checker"))
```

Comment:
why is this needed?

Reply:
I wasnt able to reference the _models from here for some reason :/ I'll look back into why this was necessary in the future